manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.3 $debug=NVS1.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: trinsic.cli
+PackageVersion: 1.2.2
+PackageLocale: en-US
+Publisher: Trinsic (Streetcred ID)
```

Comment:
```suggestion
Publisher: Trinsic
```

Reply:
This must match Control Panel.

Reply:
Will change control panel to match this, good to know, thanks!